src/Services/Flight/src/Flight/Aircrafts/Models/ValueObjects/Model.cs

```diff
@@ -0,0 +1,17 @@
+namespace Flight.Aircrafts.Models.ValueObjects;
+using System;
+public record Model : GenericValueObject<string>
```

Comment:
Please remove generic approach for this value object too.

Reply:
I'll work on implementing the recommended changes.

Reply:
done